File: CS50x/Problem_Set_1/cash.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int n;
    int num_coins = 0;

    n = get_int("Change owed: ");
    while (n < 0)
    {
        n = get_int("Change owed: ");
    }

    while (n > 24)
    {
        if (n - 25 >= 0)
            num_coins++;
        n -= 25;
    }

    while (n > 9)
    {
        if (n - 10 >= 0)
            num_coins++;
        n -= 10;
    }

    while (n > 4)
    {
        if (n - 5 >= 0)
            num_coins++;
        n -= 5;
    }

    while (n > 0)
    {
        if (n - 1 >= 0)
            num_coins++;
        n -= 1;
    }

    printf("%d\n", num_coins);
}

File: CS50x/Problem_Set_6/cash.py
while True:
    try:
        dollar = float(input("Change: "))
        if (dollar > 0.0):
            break
    except:
        print("", end="")

n = round(dollar * 100)
num_coins = 0

while n > 0:
    if n > 24:
        n -= 25
        num_coins += 1
    elif n > 9:
        n -= 10
        num_coins += 1
    elif n > 4:
        n -= 5
        num_coins += 1
    else:
        n -= 1
        num_coins += 1

print(num_coins)
